src/gpodder/__init__.py

```diff
@@ -50,8 +50,8 @@
 # This metadata block gets parsed by setup.py - use single quotes only
 __tagline__ = 'Media and podcast aggregator'
 __author__ = 'Thomas Perl <thp@gpodder.org>'
-__version__ = '4.13.0'
-__date__ = '2023-03-13'
+__version__ = '4.13.2'
+__date__ = '2023-08-30'
```

Comment:
I'm more used to tagging new versions as part of the mainline/master branch, and not as part of a PR.